Editor/Core/CoreUtils.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Linq;
+using System.Reflection;
+using UnityEditor;
+
+namespace Unity.ProjectAuditor.Editor.Core
+{
+    internal static class CoreUtils
+    {
+        public static bool HasPlatformAttribute(Type type, BuildTarget platform)
```

Comment:
Just a thought about the naming:

By design we know that _no attribute_ means we return true anyway, and the name implies that it actually has an attribute.

Maybe the method could be named "HasPlatformAttributeOrNone()" or a different wording like "SupportsPlatform" / "AppliesToPlatform".

(Naming is hard... there's always a seemingly better wording)

Reply:
Good point. I'll rename to `SupportsPlatform`

Indeed, naming is hard!